docs/containers/ldap.md

```diff
@@ -0,0 +1,29 @@
+# 👥 LDAP
+
+The LDAP sample data is based on https://github.com/rroemhild/docker-test-openldap and extended with randomly generated users/groups. For details see [data/ldap-generator/](https://github.com/juliushaertl/nextcloud-docker-dev/tree/master/data/ldap-generator). LDAP will be configured automatically if the ldap container is available during installation.
+
+
+|uid (login) | password |
+|---|---|
+| leela | leela |
+| fry | fry |
+| zoidberg | zoidberg |
+| hermes | hermes |
+| professor | professor |
+| ... | ... |
+
+
+To add LDAP in your dev environment use these commands :
+
+```bash
+docker compose down -v
```

Comment:
@juliushaertl is it safe to drop all data and reinit the server? Will all files be reinserted (`occ files:scan`)?

I would at least warn the user that this might delete user (test) data.